Xamarin.Forms.Core/BindingExpression.cs

```diff
@@ -162,7 +162,7 @@ void ApplyCore(object sourceObject, BindableObject target, BindableProperty prop
 					value = Binding.GetSourceValue(value, property.ReturnType);
 				}
 				else
-					value = property.DefaultValue;
+					value = Binding.FallbackValue ?? property.DefaultValue;
```

Comment:
should this be property.GetDefaultValue();

Reply:
I think you're right, but it's unrelated to the issue at hand. Also, while setting the default value (or even the fallback ?) we should wet the bpcontext flags accordingly.

Reply:
see #2752